gateway/types/readconfig_test.go

```diff
@@ -228,6 +228,7 @@ func TestRead_UseNATS(t *testing.T) {
 	defaults := NewEnvBucket()
 	defaults.Setenv("faas_nats_address", "nats")
 	defaults.Setenv("faas_nats_port", "6222")
+	defaults.Setenv("faas_nats_cluster_name", "example-nats-cluster")
```

Comment:
It seems like a good opportunity to improve the test to ensure we are loading the values correctly. Would you be willing to add a check on the loaded value. The code looks 💯 correct, but it will prevent potential regressions due to weird typos etc

Reply:
Which test would you like to see before merge?

Reply:
@LucasRoesler sure, I've added some :+1: